app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :likes
+  has_many :comments
```

Comment:
- [ ] Please 🙏🏽 add `foreign_key: :author_id` to the like and comments tables to join them with the user table.

Reply:
If what I said above is correct, then I guess this change too does not need to be made.

Reply:
Hello, @sameedmohsin28.

You have the reason in your point, as CR I have been reviewing according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). I hope the people in charge can change to use the same ERD diagram. For this reason I have changed the required change to OPTIONAL.

Thanks for your suggestion.

Reply:
@franclobo 

As per [Active Record lesson by Microverse](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/ActiveRecord_lesson.md#associations), and [Ruby on Rails guide](https://guides.rubyonrails.org/association_basics.html#the-types-of-associations), adding the corresponding foreign key is only required when the name of the foreign key column is set something other than the default name.
Kindly consider this as well. Thanks!